Source/ACE.Server/WorldObjects/Hook.cs

```diff
@@ -236,7 +236,7 @@ public void UpdateHookVisibility()
         {
             if (!HasItem)
             {
-                if (!(House.HouseHooksVisible ?? false))
+                if (!(House?.RootHouse?.HouseHooksVisible ?? false))
```

Comment:
I was okay with putting this around 1 place where the actual crash occurred, and maybe some logging to find out *why* that happened, and which specific house was null, however i'm not as okay with sprinkling this logic around randomly throughout the program. House and RootHouse shouldn't be null. All of this code implies that they can be null, and no explanation as to why.